stories/components/actionbar.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { ActionBar } from 'components/UI'
+
+storiesOf('Components', module).addWithChapters('ActionBar', {
+  subtitle: 'One more way to have dialog-like component',
```

Comment:
Subtitle should be updated to refer to the actionbar

Reply:
Done